src/EditorFeatures/CSharpTest/UseLocalFunction/UseLocalFunctionTests.cs

```diff
@@ -1993,6 +1993,76 @@ void Goo()
         var buildCancelled = false;
         void onUpdateSolutionCancel(int a) { buildCancelled = true; }
     }
+}");
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseLocalFunction)]
+        [WorkItem(23149, "https://github.com/dotnet/roslyn/issues/23149")]
+        public async Task TestNotAvaliableIfTypeParameterChanged1()
```

Comment:
>TestNotAvaliableIfTypeParameterChanged1 [](start = 26, length = 39)

Typo: Available (here and below)

Reply:
➡️ Fixed in 44e67c5796d99ff0dccda995063e0f1092b9cd6f